setup.py

```diff
@@ -267,6 +268,26 @@ def get_check_description(name, filterfunc):
         print("Checks templates written to %r" % (filename))
 
 
+def parse_long_description(filename):
+    readme_lines = open(
+        os.path.join(os.path.dirname(__file__), filename)
+    ).read().split("\n")
+    pootle = "Pootle"
+    notes = "Release notes"
+    initial_title = [  # PyPI doesn't like |logo|
+        pootle,
+        "="*len(pootle)
+    ]
+    release_notes = [  # Missing in README.
```

Comment:
I told you my mind does not associate "Changes" with "Release notes" :) I am undoing this.